app/Nova/Resources/Integration.php

```diff
@@ -222,7 +222,7 @@ function (Text $field, NovaRequest $request, FormData $formData) {
         }
 
         return array_merge($fields, [
-            HasMany::make('UDB3 Organizers', 'organizers', UdbOrganizer::class)
+            HasMany::make('UDB3 Organizers', 'organizers', UdbOrganizerModel::class)
```

Comment:
I think the second parameter can be `udbOrganizers` which then matches the renamed method in the `IntegrationModel`